.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
+
+# Check if the commit message has a valid sign-off.
+- repo: https://github.com/gklein/check_signoff
+  rev: v1.0.5
+  hooks:
+    - id: check-signoff
```

Comment:
commitlint checks this too, I think?

Reply:
Locally it was not checking sign-off for me, that's why I added it explicitly.